= Experiment Design
:sectnums:
:toc: left
:doctype: book
//only works on some backends, not HTML
:showcomments:
//use style like Section 1 when referencing within the document.
:xrefstyle: short
:figure-caption: Figure
:pdf-page-size: A4

//GitHub specific settings
ifdef::env-github[]
:tip-caption: :bulb:
:note-caption: :information_source:
:important-caption: :heavy_exclamation_mark:
:caution-caption: :fire:
:warning-caption: :warning:
endif::[]

The PRIDE Experiment Design project will handle and represent each PRIDE project with the corresponding experiment design, including the annotation of:

* Biological sample Metadata.
* The relationship between the Sample and the MSRun
* The Fraction and Fraction Group
* MSRun Metadata.

The following design is based on other efforts from link:external-examples/openms-experimental/OpenMS.md[OpenMS], link:external-examples/maxquant/mqpar-jarnuczak-phospho.xml[MaxQuant], link:external-examples/arrayexpress/ArrayExpress.md[ArrayExpress]

[[ontologies-supported]]
== Ontologies Supported

The list of ontologies supported by this project can be found https://github.com/PRIDE-Archive/pride-metadata-standard#3-ontologies[here].

[[sample-metadata]]
== Minimum information about Sample

The *Sample* metadata capture the steps in the chain of treatments applied to the original Source. The sample metadata structure is based on the MAGE-TAB SDRF file format developed by the https://www.ebi.ac.uk/arrayexpress/help/magetab_spec.html[RNA-Seq community].

TIP: The collaboration with the RNA-Seq community and the compatibility with the SDRF file format will enable the development of tools and bioinformatics workflows that can analyze all type of expression data (proteomics, transcriptomics).

The Sample metadata has different *Categories/Headings*  to organize all the attributes / column headers of the Sample:

|===
|Property        | Mandatory/Optional | Cardinality | Description | Example

|Source  Name    | Optional           | 1           | The Sample name. | Source Name
|Characteristics | Mandatory          | 0..*        | “Characteristics” column headings should contain an ontology property term in square brackets. The source database or file for ontology terms in these columns may be given in an adjacent “Term Source” column immediately to the right of the “Characteristics” column. In the absence of a “Term Source” column the value is assumed to be user defined. Multiple Characteristic columns of the same category (e.g., “Characteristics[Organism Part]”) are allowed. Typically the usage implies whole to part from left to right. | Characteristics [Organism Part]
|Factor Value    | Optional           | 0..*        | “Factor Value” columns should indicate which experimental factor / variable are use to perform the quantitative data analysis. The “Factor Value” columns should occur after all Characteristics and the attributes of the samples. | Factor Value [phenotype]
|===

Each Sample in the experiment MUST contains a _Source Name_, and a collection of _Characteristics_. A minimum example of a valid annotated dataset can be:

|===
| Source Name | Characteristics [Organism] | Characteristics [Organism Part] | Characteristics [phenotype] | Characteristics [phenotype] | Factor Value [phenotype]

|sample_treat   |Homo Sapiens |Whole Organism | necrotic tissue      | sample treated with drug A | necrotic tissue
|sample_control |Homo Sapiens |Whole Organism | control              | control                    |     control
|===

Some important notes:

 - Each characteristics name in the column header SHOULD be a control vocabulary term from the https://www.ebi.ac.uk/ols/ontologies/efo[EFO ontology]. For example, the Header `Characteristics [Organism]` correspond to the ontology term http://www.ebi.ac.uk/efo/EFO_0000634[Organism].

 - Multiple values for the same Characteristics are allowed. In the previous example, the `Characteristics [phenotype]` attribute is used to capture two different phenotypes conditions of the sample.

IMPORTANT: Each value in the row can be free-text or a control vocabulary term.

[[sample-templates]]
== Sample metadata templates

We have defined different experiment templates to organize the major submissions in public proteomics databases:

- Human: All tissue-based experiments that use Human samples.
- Vertebrates: Vertebrate experiment.
- Non-vertebrates: Non-vertebrate experiment.
- Plants: Plant experiment.
- Cell lines: Experiments using cell-lines.
- Other

*Sample attributes*: Minimum sample attributes for primary cells from different species and cell lines.

|===
| |Human |Vertebrates |Non-vertebrates |Plants |Cell lines |Comment 

|Organism            |X |X |X |X |X |
|Strain/breed        | |O |O |O |O |
|Ethnicity           |X |O |O |O |X |
|Age |O |O | | | |
|Developmental stage |X |O | | | | 
|Sex |X |O | | | | 
|Disease |X |X |X |X |X | 
|Organism part |X |X |X |X |X | 
|Cell type* |X |X |X |X |X |* if known, see comment below 
|Individual |O |O |O |O | |donor or animal ID 
|Cell line Code | | | | |X |name of commercial cell line 
|===

* X : Required Attributes for each sample Type (e.g. Human, Vertebrates).
* O : Optional Attributes.

X: Required
O: Not Required

== Annotation MSRun

MSRun: From each MSRun we need to capture:
 - *unique identifier* : A unique identifier provided by PRIDE during the submission process.
 - *filename*: The filename provided by the user during the submission process.

== Relationship between Sample, Fraction and MSRun

The relationship between the Sample and MSRun will be given by two other Properties:

* Label: It represents the information of a Multiplex or label-free experiment. It can be used to spot the type of experiment. For each Label two major characteristics are mandatory:

** Label Identifier: An identifier of the type of labeling in the Experiment Design Table.
** Label Name: A CvTerm with the type of Labeling applied: *["PRIDE", "PRIDE:0000312", "Label free", ]*
* Fraction Id

** Fraction Identifier: A unique fraction identifier to classify the sample.

*Additional*:

Some search engines as MaxQuant use the Fraction Group to perform better statistical analysis:

* Fraction Group
** Fraction Group Identifier: The fraction group identifier is used to classify multiple fractions.

*Examples Label Free Experiment*:

