= Experiment Design
:sectnums:
:toc: left
:doctype: book
//only works on some backends, not HTML
:showcomments:
//use style like Section 1 when referencing within the document.
:xrefstyle: short
:figure-caption: Figure
:pdf-page-size: A4

//GitHub specific settings
ifdef::env-github[]
:tip-caption: :bulb:
:note-caption: :information_source:
:important-caption: :heavy_exclamation_mark:
:caution-caption: :fire:
:warning-caption: :warning:
endif::[]

The Proteomics Experimental Design project will handle and represent each proteomics dataset with the corresponding experimental design, including the annotation of:

* Biological sample metadata.
* The relationships between the Samples and the MSRuns
* MSRun (technical) Metadata.

The following design is based on other efforts from link:../old-discussions/proteomics-propietary-examples/external-examples/openms-experimental/OpenMS.md[OpenMS], link:../old-discussions/proteomics-propietary-examples/external-examples/maxquant/mqpar-jarnuczak-phospho.xml [MaxQuant], and link:../old-discussions/proteomics-propietary-examples/external-examples/arrayexpress/ArrayExpress.md[ArrayExpress]

[[ontologies-supported]]
== Ontologies/Controlled Vocabularies Supported

The list of ontologies/controlled vocabularies supported can be found https://github.com/bigbio/pride-metadata-standard#3-ontologies[here].

[[sdrf-file-format]]
== The sample and data relationship format

The https://www.ebi.ac.uk/arrayexpress/help/creating_a_sdrf.html[SDRF] (Sample and Data Relationship Format) describes the sample characteristics and the relationships between samples and  data files, etc. The information in SDRF files is organised so that it follows the natural flow of a functional genomics experiment, since it was originally developed for this type of experiments. It begins by describing the samples and finishes with the names of the data files generated from the analyses of the experimental results. Here we are introducing and adapting the SDRF file format for the proteomics community.

The SDRF file is a tab-delimited format where each ROW corresponds to a relation between a _Sample_ and a _Data_ file, each column corresponds to an attribute/property of the _Sample_ and the value in each cell is the specific *value* of the property for a given _Sample_.

NOTE: We expect that the collaboration with the transcriptomics and functional genomics community through the existing SDRF file format will enable the development of tools and bioinformatics workflows that can analyze together different types of expression data (proteomics and transcriptomics).

SDRF for proteomics in a Nutshell
[#img-sunset]
image::https://github.com/bigbio/proteomics-metadata-standard/raw/master/experimental-design/images/sdrf-nutshell.png[SDRF for proteomics in Nutshell]

[[sdrf-file-standarization]]
=== SDRF values

- Unknown values: In some cases the column is Mandatory (:white_check_mark:) but for some samples the value is unknown. In those cases users SHOULD use **not available**.
- Not Applicable values: In some cases the column is Mandatory (:white_check_mark:) but for some samples the value is not applicable: In those cases users SHOULD use **not applicable**.
- Case sensitive: By specification the SDRF is case insensitive, but we RECOMMEND to use lower cases throughout all the text (Column names and values).
- Spaces: By specification the SDRF is insensitive to spaces (SourceName == source name). We RECOMMENDED to use the space representation because is more human readable (e.g source name).

[[sample-metadata]]
=== Minimum information about Samples

The first column of the SDRF is the **Source Name** (a unique accession of the sample, example - Sample 1). The *Sample* metadata captures the properties of the sample and they are defined as *characteristics*. The sample metadata structure is based on the MAGE-TAB SDRF file format developed by the http://fged.org/projects/mage-tab/[RNA-Seq community].

The Sample metadata has different *Categories/Headings*  to organize all the attributes / column headers of a given Sample:

|===
|Property        | Mandatory(:white_check_mark:)/Optional(:zero:) | Cardinality | Description | Example

|source  name    | :white_check_mark:             | 1           | The Sample name. | Sample 1
|characteristics | :white_check_mark: | 0..*      | “characteristics” column headings should contain an ontology property term in square brackets.*

Multiple _Characteristic_ columns of the same category (e.g., “characteristics[organism part]”) are allowed. Typically the usage implies whole to part from left to right. | characteristics [Organism Part]
|===

Each Sample in the experiment MUST contains a _Source Name_, and a collection of _characteristics_:

|===
| source name | characteristics[organism] | characteristics[organism part] | characteristics[compound] | characteristics[phenotype] | factor value[phenotype]

|sample_treat   |homo sapiens |Whole Organism | necrotic tissue      | compound: drug A | necrotic tissue
|sample_control |homo sapiens |Whole Organism | normal               | compound: none   | normal
|===

Some important notes:

 - Each _characteristics_ name in the column header SHOULD be a control vocabulary (CV) term from the https://www.ebi.ac.uk/ols/ontologies/efo[EFO ontology]. For example, the Header `characteristics[organism]` corresponds to the ontology term http://www.ebi.ac.uk/efo/EFO_0000634[Organism].
 - Multiple values (columns) for the same _characteristics_ term are allowed in the SDRF. However, we RECOMMENDED not to use same column in the same SDRF. If you have multiple phenotypes, you can specify what it refers to or use another more specific term, e.g. "immunophenotype".

NOTE: Each value in the row can be free-text or a control vocabulary term. In the previous example, the `characteristics[phenotype]` value CAN BE the free-text _control_ or the corresponding ontology identifier in https://www.ebi.ac.uk/ols/ontologies/efo[EFO] _http://www.ebi.ac.uk/efo/EFO_0001461_

[[sdrf-values-properties]]
=== SDRF values for each property

The value for each property (e.g. _characteristics_, _comment_) corresponding to each sample can be represented in multiple ways.

- Free Text (Human readable): In the free text representation, the value is provided as text without Ontology support (e.g. colon, or providing accession numbers). This is only RECOMMENDED when the **text** inserted in the table is the exact _name_ of an ontology/CV term in EFO.

|===
| source name | characteristics[organism]

| sample 1 |homo sapiens
| sample 2 |homo sapiens
|===

- Ontology url (Computer readable): Users can provide the corresponding URI of the ontology/CV term as a value. This is recommended for enriched files where the client does not want to use intermediate tools to map from Free Text to ontology/CV terms.

|===
| source name | characteristics[organism]

| Sample 1 |http://purl.obolibrary.org/obo/NCBITaxon_9606
| Sample 2 |http://purl.obolibrary.org/obo/NCBITaxon_9606
|===

- Key=value representation (Human and Computer readable): The current representation aims to provide a mechanism to represent the complete information of the ontology/CV term including _Accession_, _Name_ and other additional properties (see example, <<encoding-protein-modifications>>).

  In the key=value pair representation the Value of the property is represented as an Object with multiple properties where the key is one of the properties of the object and the value is the corresponding value for the particular key. For example:
  NM=Glu->pyro-Glu; MT=fixed; PP=Anywhere; AC=Unimod:27; TA=E


[[from-sample-scan]]
== From Samples to Assay (MSRun)

The connection between the _Sample_ to the final Assay (_MSrun_) is done by using a series of properties and attributes. All the properties needed to relate a given _Sample_ to the corresponding _MsRun_ are annotated with the category *comment*. The use of _comment_ is mainly aimed at differentiating Sample _characteristics_ from the MSrun properties. The following properties SHOULD be provided for each Sample/MSRun:

- assay name: For SDRF compatibilities we can not use MSRun but _assay name_. Examples of assay name: run 1, run_fracion_1_2

- comment[fraction identifier]: The _fraction identifier_ allows to record the number of a given fraction. The fraction identifier corresponds to this https://www.ebi.ac.uk/ols/ontologies/ms/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FMS_1000858[ontology term].

- comment[label]: The _label_ describes the label applied to each Sample (if any). In case of multiplex experiments such as TMT, SILAC, and/or ITRAQ the corresponding _label_ SHOULD be added. For Label-free experiments the https://www.ebi.ac.uk/ols/ontologies/ms/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FMS_1002038[label free sample] term MUST be added.

- comment[data file]: The _data file_ provides the name of the raw file from the instrument. The raw data file corresponds to this https://www.ebi.ac.uk/ols/ontologies/ms/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FMS_1000577[ontology term].

NOTE: the order of the columns are important, **assay name** SHOULD we always before the comments. We RECOMMENDED to put the last column as comment[data file]
|===
|        |  assay name      | comment[label]    | comment[fraction identifier] | comment[data file]
|sample 1|  run 1           | label free sample | 1                            | 000261_C05_P0001563_A00_B00K_R1.RAW
|sample 1|  run 2           | label free sample | 2                            | 000261_C05_P0001563_A00_B00K_R2.RAW
|===

TIP: All the possible _label_ values can be seen in the in the PSI-MS CV under the https://www.ebi.ac.uk/ols/ontologies/ms/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FMS_1002602[sample label] node.

In the case that technical and/or biological replicates have been measured, this information is not sufficient anymore.
To be able to trace a given quantitative value to the exact replicate one needs encode this information as part of the experimental design.

In the following example, only if the technical replicate column is provided, one can distinguish quantitative values of the same fraction but different technical replicates.

|===
| source name       | assay name | comment[label]    | comment[fraction identifier] | comment[technical replicate] | comment[data file]
| Sample 1          |    run 1   | label free sample | 1                            | 1                            | 000261_C05_P0001563_A00_B00K_F1_TR1.RAW
| Sample 1          |    run 2   | label free sample | 2                            | 1                            |  000261_C05_P0001563_A00_B00K_F2_TR1.RAW
| Sample 1          |    run 3   | label free sample | 1                            | 2                            | 000261_C05_P0001563_A00_B00K_F1_TR2.RAW
| Sample 1          |    run 4   | label free sample | 2                            | 2                            | 000261_C05_P0001563_A00_B00K_F2_TR2.RAW
|===


The “comment” columns in *SDRF* are included as a basic extensibility mechanism for local implementations. The name associated with the comment is included in square brackets in the column heading, and the value(s) entered in the body of the column. _comment_ columns could be used in various ways - to provide references to external files like raw files, or to include identifiers of objects in external systems.

[[additional-raw-file]]
=== Additional RAW file properties

We RECOMMEND to include the public URI of the file if available. For example for PRIDE datasets the URI from the FTP can be provided:

|===
|   |... |comment[associated file uri]

|sample 1| ... |ftp://ftp.pride.ebi.ac.uk/pride/data/archive/2017/09/PXD005946/000261_C05_P0001563_A00_B00K_R1.RAW
|===

[[sample-scan-additional]]
=== MSRun additional properties

Some analysis software as MaxQuant use the 'Fraction Group' concept to perform better statistical analysis:

- comment[fraction group]: Some quantitative tools use the 'Fraction Group' to know which fractions belong together. In MaxQuant the 'Fraction Group' is called "Experiment".


[[encoding-MSRun-technical-details]]
== MSRun technical details properties

We RECOMMEND to encode some of the technical parameters of the MS experiment as _comment_s (https://www.ebi.ac.uk/arrayexpress/help/creating_a_sdrf.html[Check what is a comment in SDRF]) including the following parameters:

- Protein Modifications <<encoding-protein-modifications>>
- Precursor and Fragment mass tolerances <<encoding-tolerances>>
- Digestion Enzyme <<encoding-enzymes>>

[[encoding-protein-modifications]]
=== Protein Modifications

Sample modifications (including both chemical modifications and post translational modifications, PTMs) are originated from multiple sources: **artifacts modifications**, **isotope labeling**, adducts that are encoded as PTMs (e.g . sodium) or the most **biologically relevant** PTMs. The most common and widely studied PTMs include phosphorylation and glycosylation, among many others. Many of these PTMs are critical to a given protein's function.

The current specification RECOMMENDS to provide Sample modifications including the aminoacid affected, if is Variable or Fixed (also Custom and Annotated modifications are supported) and other properties such as mass shift/delta mass and the position (e.g. anywhere in the sequence).

The RECOMMENDED name of the column for sample modification parameters is:

  comment[modification parameters]

NOTE: The `modification parameters` is the name of the ontology term https://www.ebi.ac.uk/ols/ontologies/ms/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FMS_1001055[MS:1001055]

For each modification, we will capture different properties in a `key=value` pair structure including name, position, etc. All the possible features available for modification parameters:

|===
|Property |Key |Example | Mandatory(:white_check_mark:)/Optional(:zero:) |comment

|Name of the Modification| NM | NM=Acetylation | :white_check_mark: | * Name of the Modification, for custom modifications can be a name defined by the user.
|Modification Accession  | AC | AC=UNIMOD:1    | :zero:             | Accession in an external database UNIMOD or PSI-MOD supported.
|Chemical Formula        | CF | CF=H(2)C(2)O   | :zero:             | This is the chemical formula of the added or removed atoms. For the formula composition please follow the guidelines from http://www.unimod.org/names.html[UNIMOD]
|Modification Type       | MT | MT=Fixed       | :zero: | This specifies which modification group the modification should be included with. Choose from the following options: [Fixed, Variable, Annotated]. _Annotated_ is used to search for all the occurrences of the modification into an annotated protein database file like UNIPROT XML or PEFF.
|Position of the modification in the Polypeptide |  PP | PP=Any N-term | :white_check_mark: | Choose from the following options: [Anywhere, Protein N-term, Protein C-term, Any N-term, Any C-term]
|Target Amino acid       | TA | TA=S,T,Y       | :white_check_mark: | The target amino acid letter. If the modification targets multiple sites, it can be separated by `,`.
|Monoisotopic Mass       | MM | MM=42.010565   | :zero: | The exact atomic mass shift produced by the modification. Please use at least 5 decimal places of accuracy. This should only be used if the chemical formula of the modification is not known. If the chemical formula is specified, the monoisotopic mass will be overwritten by the claculated monoisotopic mass.
|Target Site             | TS | TS=N[^P][ST]   | :zero: | For some software, it is important to capture complex rules for modification sites as regular expressions. These use cases should be specified as regular expressions.
|===


NOTE: We RECOMMEND to use for the modification name the UNIMOD interim name or the PSI-MOD name. For custom modifications, we RECOMMEND to use an intuitive name. If the PTM is unknown (custom), the _Chemical Formula_ or _Monoisotopic Mass_ MUST be annotated.

An example of a **SDRF** file with sample modifications annotated:

|===
| |comment[modification parameters] | comment[modification parameters]

|sample 1| NM=Glu->pyro-Glu; MT=fixed; PP=Anywhere; AC=Unimod:27; TA=E | NM=Oxidation; MT=Variable; TA=M
|===

[[encoding-enzymes]]
=== Enzyme

The `comment [cleavage agent details]` property is used to capture the Enzyme information. Similar to protein modification <<encoding-protein-modifications>> we will use a key=value pair representation to encode the following properties for each enzyme:

|===
|Property           |Key |Example     | Mandatory(:white_check_mark:)/Optional(:zero:) | comment
|Name of the Enzyme | NE | NM=Trypsin | :white_check_mark:                             | * Name of the Enzyme.
|Enzyme Accession | AC | AC=MS:1001251 | :zero:                                      | Accession in an external PSI-MS Ontology definition under the following category https://www.ebi.ac.uk/ols/ontologies/ms/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FMS_1001045[Cleavage agent name].
|Cleavage site regular expression | CS | CS=(?<=[KR])(?!P) | :zero: | The cleavage site defined as a regular expression.
|===

An example of a **SDRF** with sample enzyme annotated:

|===
| |comment[cleavage agent details]

|sample 1| NE=Trypsin; AC=MS:1001251; CS=(?<=[KR])(?!P)
|===


[[encoding-tolerances]]
=== Precursor and Fragment mass tolerances

Encoding precursor and fragment tolerances, for proteomics experiments is important to encode different tolerances (Precursor and fragment).

|===
| |comment[fragment mass tolerance]	| comment[precursor mass tolerance]

|sample 1| 0.6 Da |	20 ppm
|===


[[label-annotatations]]
=== Label annotations

In order to annotate quantitative projects, the SDRF file format use tags for each channel associated with the sample in comment[label].

Some of the most popular labels are:

* For label-free experiments the value should be: label-free
* For TMT experiments the SDRF uses the PRIDE ontology terms under sample label. Here some examples of TMT channels:
** TMT126, TMT127, TMT127C , TMT127N, TMT128 , TMT128C, TMT128N, TMT129, TMT129C, TMT129N, TMT130, TMT130C, TMT130N, TMT131



[[sdrf-templates]]
== Sample/MSRun templates

The *sample metadata templates* are a set of guidelines to annotate different type of proteomics experiments to ensure that a Minimum Metadata and `characteristics` are provided to understand the dataset. These templates respond to the distribution and frequency of experiment types in public databases like http://www.ebi.ac.uk/pride/archive[PRIDE] and http://www.proteomexchange.org/[ProteomeXchange]:

- Default: Minimum information for any proteomics experiment https://github.com/bigbio/proteomics-metadata-standard/blob/master/templates/sdrf-default.txt[Template]
- Human: All tissue-based experiments that use Human samples https://github.com/bigbio/proteomics-metadata-standard/blob/master/templates/sdrf-human.txt[Template]
- Vertebrates: Vertebrate experiment. https://github.com/bigbio/proteomics-metadata-standard/blob/master/templates/sdrf-vertebrates.txt[Template]
- Non-vertebrates: Non-vertebrate experiment. https://github.com/bigbio/proteomics-metadata-standard/blob/master/templates/sdrf-nonvertebrates.txt[Template]
- Plants: Plant experiment. https://github.com/bigbio/proteomics-metadata-standard/blob/master/templates/sdrf-plants.txt[Template]
- Cell lines: Experiments using cell-lines. https://github.com/bigbio/proteomics-metadata-standard/blob/master/templates/sdrf-cell-line.txt[Template]

*Sample attributes*: Minimum sample attributes for primary cells from different species and cell lines

|===
|                                       | Default            |Human              | Vertebrates       | Non-vertebrates   | Plants            | Cell lines
|Source Name                            | :white_check_mark: |:white_check_mark: |:white_check_mark: |:white_check_mark: |:white_check_mark: |:white_check_mark:
|characteristics[organism]              | :white_check_mark: |:white_check_mark: |:white_check_mark: |:white_check_mark: |:white_check_mark: |:white_check_mark:
|characteristics[strain/breed]          |                    |                   |                   |:zero:             |                   |:zero:
|characteristics[ecotype/cultivar]      |                    |                   |                   |                   |:zero:             |
|characteristics[ethnicity]             |                    |:white_check_mark: |                   |                   |                   |
|characteristics[age]                   |                    |:white_check_mark: |:zero:             |                   |:zero:             |
|characteristics[developmental stage]   |                    |:white_check_mark: |:zero:             |                   |:zero:             |
|characteristics[sex]                   |                    |:white_check_mark: |:zero:             |                   |                   |
|characteristics[disease]               | :white_check_mark: |:white_check_mark: |:white_check_mark: |:white_check_mark: |                   |:white_check_mark:
|characteristics[organism part]         | :white_check_mark: |:white_check_mark: |:white_check_mark: |:white_check_mark: |:white_check_mark: |:white_check_mark:
|characteristics[cell type]             | :white_check_mark: |:white_check_mark: |:white_check_mark: |:white_check_mark: |:white_check_mark: |:white_check_mark:
|characteristics[individual]            |                    |:zero:             |:zero:             |:zero:             |:zero:             |:zero:
|characteristics[cell line Code]        |                    |                   |                   |                   |                   |:white_check_mark:
|                                       |                    |                   |                   |                   |                   |
|comment[data file]                     | :white_check_mark: |:white_check_mark: |:white_check_mark: |:white_check_mark: |:white_check_mark: |:white_check_mark:
|comment[fraction identifier]           | :white_check_mark: |:white_check_mark: |:white_check_mark: |:white_check_mark: |:white_check_mark: |:white_check_mark:
|comment[label]                         | :white_check_mark: |:white_check_mark: |:white_check_mark: |:white_check_mark: |:white_check_mark: |:white_check_mark:

|===

* :white_check_mark: : Required Attributes for each sample Type (e.g. Human, Vertebrates).
* :zero: : Optional Attribute


[[sdrf-factor-value]]
== SDRF study variable

The variable/property under study should be highlighted using the *factor value* category. For example, the **factor value[disease]** is used when the user wants to compare expression across different diseases.

|===
|factor value    | :zero:           | 0..*        | “factor value” columns should indicate which experimental factor / variable are use to perform the quantitative data analysis. The “factor value” columns should occur after all characteristics and the attributes of the samples. | Factor Value [phenotype]
|===


NOTE: The factor value[_property_] is optional (:zero:) because it depend of the analysis that the user wants to perform with the sample. For example, the original submitter of the dataset probably studied the phenotype variable but the reanalysis is focus on cell line.

[[compose-sdrf-files]]
== Multiple projects into one annotation file

PRIDE curators can decide to annotate multiple PRIDE Projects into one big sdrf for reanalysis purpose. If that is the case, we RECOMMENDED to use the __comment[proteomexchange accession number]__ to differentiate between projects.

