= Experiment Design
:sectnums:
:toc: left
:doctype: book
//only works on some backends, not HTML
:showcomments:
//use style like Section 1 when referencing within the document.
:xrefstyle: short
:figure-caption: Figure
:pdf-page-size: A4

//GitHub specific settings
ifdef::env-github[]
:tip-caption: :bulb:
:note-caption: :information_source:
:important-caption: :heavy_exclamation_mark:
:caution-caption: :fire:
:warning-caption: :warning:
endif::[]

The PRIDE Experiment Design project will handle and represent each PRIDE project with the corresponding experiment design, including the annotation of:

* Biological sample metadata.
* The relationship between the Sample and the MSRun
* MSRun Metadata.

The following design is based on other efforts from link:../proteomics-propietary-examples/external-examples/openms-experimental/OpenMS.md[OpenMS], link:../proteomics-propietary-examples/external-examples/maxquant/mqpar-jarnuczak-phospho.xml[MaxQuant], link:../proteomics-propietary-examples/external-examples/arrayexpress/ArrayExpress.md[ArrayExpress]

[[ontologies-supported]]
== Ontologies Supported

The list of ontologies supported by this project can be found https://github.com/PRIDE-Archive/pride-metadata-standard#3-ontologies[here].

[[sdrf-file-format]]
== The sample and data relationship format

The https://www.ebi.ac.uk/arrayexpress/help/creating_a_sdrf.html[SDRF] (Sample and Data Relationship Format) describes the sample characteristics and the relationship between samples, and  data files etc. The information in the SDRF is organised so that it follows the natural flow of a functional genomics experiment. It begins with describing your samples and finishes with the names of the data files generated from the analysis of the experiment results. In the current project, we are introducing and adapting the SDRF file format to the proteomics community.

NOTE: The collaboration with the RNA-Seq community and the compatibility with the SDRF file format will enable the development of tools and bioinformatics workflows that can analyze all type of expression data (proteomics, transcriptomics).

The SDRF file is a tab-delimited format where each ROW correspond to a _Sample_ Each, each column correspond to an attribute/property of the _Sample_ and the value in each cell is the specific *value* of the property for a given Sample.


|===
|Source Name |Characteristics[organism] |Characteristics[individual] |Characteristics[developmental stage]
|sample 1	 |Homo sapiens	            |individual 58	             |adult
|sample 2    |Homo sapiens	            |individual 52	             |adult
|===



[[sample-metadata]]
== Minimum information about Sample

The *Sample* metadata capture the steps in the chain of treatments applied to the original Source. The sample metadata structure is based on the MAGE-TAB SDRF file format developed by the https://www.ebi.ac.uk/arrayexpress/help/magetab_spec.html[RNA-Seq community].


The Sample metadata has different *Categories/Headings*  to organize all the attributes / column headers of the Sample:

|===
|Property        | Mandatory/Optional | Cardinality | Description | Example

|Source  Name    | Optional           | 1           | The Sample name. | Source Name
|Characteristics | Mandatory          | 0..*        | “Characteristics” column headings should contain an ontology property term in square brackets. The source database or file for ontology terms in these columns may be given in an adjacent “Term Source” column immediately to the right of the “Characteristics” column. In the absence of a “Term Source” column the value is assumed to be user defined. Multiple Characteristic columns of the same category (e.g., “Characteristics[Organism Part]”) are allowed. Typically the usage implies whole to part from left to right. | Characteristics [Organism Part]
|Factor Value    | Optional           | 0..*        | “Factor Value” columns should indicate which experimental factor / variable are use to perform the quantitative data analysis. The “Factor Value” columns should occur after all Characteristics and the attributes of the samples. | Factor Value [phenotype]
|===

Each Sample in the experiment MUST contains a _Source Name_, and a collection of _Characteristics_. A minimum example of a valid annotated dataset can be:

|===
| Source Name | Characteristics [Organism] | Characteristics [Organism Part] | Characteristics [phenotype] | Characteristics [phenotype] | Factor Value [phenotype]

|sample_treat   |Homo Sapiens |Whole Organism | necrotic tissue      | sample treated with drug A | necrotic tissue
|sample_control |Homo Sapiens |Whole Organism | control              | control                    |     control
|===

Some important notes:

 - Each characteristics name in the column header SHOULD be a control vocabulary term from the https://www.ebi.ac.uk/ols/ontologies/efo[EFO ontology]. For example, the Header `Characteristics [Organism]` correspond to the ontology term http://www.ebi.ac.uk/efo/EFO_0000634[Organism].

 - Multiple values for the same Characteristics are allowed. In the previous example, the `Characteristics [phenotype]` attribute is used to capture two different phenotypes conditions of the sample.

NOTE: Each value in the row can be free-text or a control vocabulary term. In the previous example, the `Characteristics [phenotype]` value CAN BE the free-text _control_ or the corresponding ontology identifier in https://www.ebi.ac.uk/ols/ontologies/efo[EFO] _http://www.ebi.ac.uk/efo/EFO_0001461_

[[sample-templates]]
=== Sample metadata templates

The *sample metadata templates* are a set of guidelines to annotate different type of proteomics experiments to ensure that a Minimum Metadata and `Characteristics` are provided to understand the dataset. These templates respond to the distribution and frequency of expeirment types in public databases like http://www.ebi.ac.uk/pride/archive[PRIDE] and http://www.proteomexchange.org/[ProteomeXchange]:

- Human: All tissue-based experiments that use Human samples.
- Vertebrates: Vertebrate experiment.
- Non-vertebrates: Non-vertebrate experiment.
- Plants: Plant experiment.
- Cell lines: Experiments using cell-lines.

*Sample attributes*: Minimum sample attributes for primary cells from different species and cell lines

|===
|                                        | Human          | Vertebrates | Non-vertebrates | Plants | Cell lines | Comment

|Characteristics [Organism]              | M              |M            |M                |M       |M           |
|Characteristics [Strain/breed]          |                |O            |O                |O       |O           |
|Characteristics [Ethnicity]             | M              |O            |O                |O       |M           |
|Characteristics [Age]                   | O              |O            |                 |        |            |
|Characteristics [Developmental stage]   | M              |O            |                 |        |            |
|Characteristics [Sex]                   | M              |O            |                 |        |            |
|Characteristics [Disease]               | M              |M            |M                |M       |M           |
|Characteristics [Organism part]         | M              |M            |M                |M       |M           |
|Characteristics [Cell type] *           | M              |M            |M                |M       |M           |* if known, see comment below
|Characteristics [Individual]            | O              |O            |O                |O       |            |donor or animal ID
|Characteristics [Cell line Code]        |                |             |                 |        |M           |name of commercial cell line
|===

* M : Required Attributes for each sample Type (e.g. Human, Vertebrates).
* O : Optional Attributes.

[[from-sample-scan]]
== From Sample to MsRun

The connection from the _Sample_ to the final _MSrun_ would be done by a series of properties and attributes. We RECOMMENDED the following properties to be present on each Sample Row:

MSRun: From each MSRun we need to capture:

 - Label: The _label_ describe the labeling process applied to each Sample. In case of Multiplex Experiments such as TMT, SILAC, ITRAQ the corresponding Label should be added. For Label-free experiments the https://www.ebi.ac.uk/ols/ontologies/ms/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FMS_1002038[label free sample]

 - Comment [Fraction Identifier]: The _fraction identifier_ allows to know the number of your fraction. The fraction identifier correspond to this https://www.ebi.ac.uk/ols/ontologies/ms/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FMS_1000858[ontology term].

 - Comment [Raw Data File]: The _raw data file_ provides the name of the raw file from the instrument. The raw data file correspond to this https://www.ebi.ac.uk/ols/ontologies/ms/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FMS_1000577[ontology term].

TIP: All the possible _Label_ values can be seen in the in the PSI-MS ontology under https://www.ebi.ac.uk/ols/ontologies/ms/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FMS_1002602[sample label] node.

The “Comment” columns in *SDRF* are included as a basic extensibility mechanism for local implementations. The name associated with the comment is included in square
brackets in the column heading, and the value(s) entered in the body of the column. Comment columns could be used in various ways - to provide references to external files like
raw files or protocols; to include identifiers of objects in external systems; to qualify the type of Protocol REF (e.g., growth protocol).

[[sample-scan-additional]]
=== MSRun additional properties

Some search engines as MaxQuant use the Fraction Group to perform better statistical analysis:

- Comment [Fraction Group]: Some Quantitative tools use the Fraction Group to know which fractions belong together. In MaxQuant the Fraction Group is called "Experiment".


[[encoding-MSRun-technical-details]]
== MSRun technical details properties

We RECOMMEND to encode some of the technical parameters of the mass spectrometry experiment as Comments (https://www.ebi.ac.uk/arrayexpress/help/creating_a_sdrf.html[Check what is a Comment in SDRF]) including the following parameters:

- Protein Modifications <<encoding-protein-modifications>>
- Precursor and Fragment mass tolerances <<encoding-tolerances>>
- Enzyme <<encoding-enzymes>>

[[encoding-protein-modifications]]
=== Protein Modifications

Sample modifications (PTMs) are originated from multiple sources: **technical modifications**, **isotope labeling** or the most **biologically relevant** the wide variety of chemical modifications after translation. The most common and widely studied post translational modifications include phosphorylation and glycosylation. Many of these post-translational modifications are critical to the protein's function.

The current specification RECOMMEND to provide Sample modifications including the Amino acid affected, if is Variable or Static (Fixed) and other properties such as mass shift and position (e.g. anywhere in the sequence).

The RECOMMENDED name of the column for sample modification parameters is:

  Commet [modification parameters]

.. NOTE: The `modification parameters` is the name of the ontology term https://www.ebi.ac.uk/ols/ontologies/ms/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FMS_1001055[MS:1001055]

For each modification, we will capture different properties in a `key=value` pair structure including name, position, etc. All the possible features available for modification parameters:


|===
|Property |Key |Example | Required/Optional |Comment

|Name of the modification| NM | NM=Acetylation | Required | Name of the modification, for custom modifications can be a name defined by the user.

|Database Accession| AC | AC=UNIMOD:1 | Optional | Accession in an external database UNIMOD or PSI-MOD supported.

|Chemical Formula  | CF | CF=H(2)C(2)O| Optional | This is the chemical formula of the added or removed atoms. For the formula composition please follow the guidelines from http://www.unimod.org/names.html[UNIMOD]

|Modification type | MT | MT=Fixed | Required | This specifies which modification group the modification should be included with. Choose from the following options: [Fixed, Variable, Custom].

|Position of the modification in the polypeptide |  PP | PP=Any N-term | Required | Choose from the following options: [Anywhere, Protein N-term, Any N-term]

|Target Amino acid| TA | TA=N-term | Optional | The target amino acid letter. If the modification target multiple sites, it should be provided as Target Regular Expression (TR)

|Monoisotopic mass| MM | MM=42.010565 | Optional | The exact atomic mass shift produced by the modification. Please use at least 5 decimal places of accuracy. This will override the monoisotopic mass described in the chemical formula because there are cases where the mass of the mod and the mass shift from the mod are different (e.g. trimethylation has mass of 43 but mass shift from trimethylation is 42).

|Target regular expression | TR | Pending | Optional | For some softwares is more interesting to capture complex rules for modification sites. This use cases should be specified as regular expressions.
|===



Some examples of SDRF with sample modifications annotated:


|===
| |comment [modification parameters] | comment [modification parameters]

|sample 1| NM=Glu->pyro-Glu; MT=fixed; PP=Anywhere; AC=Unimod:27; TA=E | NM=Oxidation; MT=Variable; TA=M
|===



[[encoding-tolerances]]
=== Precursor and Fragment mass tolerances

Pending

[[encoding-enzymes]]
=== Enzyme

Pending