= Experiment Design
:sectnums:
:toc: left
:doctype: book
//only works on some backends, not HTML
:showcomments:
//use style like Section 1 when referencing within the document.
:xrefstyle: short
:figure-caption: Figure
:pdf-page-size: A4

//GitHub specific settings
ifdef::env-github[]
:tip-caption: :bulb:
:note-caption: :information_source:
:important-caption: :heavy_exclamation_mark:
:caution-caption: :fire:
:warning-caption: :warning:
endif::[]

The PRIDE Experiment Design project will handle and represent each PRIDE project with the corresponding experiment design, including the annotation of:

* Biological sample Metadata.
* The relationship between the Sample and the MSRun
* The Fraction and Fraction Group
* MSRun Metadata.

The following design is based on other efforts from link:external-examples/openms-experimental/OpenMS.md[OpenMS], link:external-examples/maxquant/mqpar-jarnuczak-phospho.xml[MaxQuant], link:external-examples/arrayexpress/ArrayExpress.md[ArrayExpress]

[[ontologies-supported]]
== Ontologies Supported

The list of ontologies supported by this project can be found https://github.com/PRIDE-Archive/pride-metadata-standard#3-ontologies[here].

[[sample-metadata]]
== Minimum information about Sample

The *Sample* metadata capture the steps in the chain of treatments applied to the original Source. The sample metadata structure is based on the MAGE-TAB file format developed by the https://www.ebi.ac.uk/arrayexpress/help/magetab_spec.html[RNA-Seq community].

TIP: The collaboration with the RNA-Seq community and the compatibility with the MAGE-TAB file format will enable the development of tools and bioinformatics workflows that can analyze all type of expression data (proteomics, transcriptomics).

|===
|Property | Mandatory/Optional | Cardinality
|Sample Name     | Optional  | 1
|Sample ID       | Mandatory | 1
|Characteristics | Mandatory | 0..*
|===

Each Sample in the experiment MUST contains a _Sample ID_, and a collection of _Characteristics_. In addition, the a Sample Name can be provided. The _Sample ID_ MUST be unique for each sample.

The following example presents a Human dataset:

|===
|Sample ID | Sample Name | Characteristics [Organism] | Characteristics [Organism Part]

|1 |sample_treat   |Homo Sapiens |Whole Organism
|2 |sample_control |Homo Sapiens |Whole Organism
|===

Each characteristics name in the column header SHOULD be a control vocabulary term from the https://www.ebi.ac.uk/ols/ontologies/efo[EFO ontology].


[[experiment-templates]]
== Experiment templates

We have defined different experiment templates to organize the major submissions in public proteomics databases:

- Human: All tissue-based experiments that use Human samples.
- Vertebrates: Vertebrate experiment.
- Non-vertebrates: Non-vertebrate experiment.
- Plants: Plant experiment.
- Cell lines: Experiments using cell-lines.
- Other

*Sample attributes*: Minimum sample attributes for primary cells from different species and cell lines.

|===
| |Human |Vertebrates |Non-vertebrates |Plants |Cell lines |Comment 

|Organism            |X |X |X |X |X |
|Strain/breed        | |O |O |O |O |
|Ethnicity           |X |O |O |O |X |
|Age |O |O | | | |
|Developmental stage |X |O | | | | 
|Sex |X |O | | | | 
|Disease |X |X |X |X |X | 
|Organism part |X |X |X |X |X | 
|Cell type* |X |X |X |X |X |* if known, see comment below 
|Individual |O |O |O |O | |donor or animal ID 
|Cell line Code | | | | |X |name of commercial cell line 
|===

* X : Required Attributes for each sample Type (e.g. Human, Vertebrates).
* O : Optional Attributes.

X: Required
O: Not Required

== Annotation MSRun

MSRun: From each MSRun we need to capture:
 - *unique identifier* : A unique identifier provided by PRIDE during the submission process.
 - *filename*: The filename provided by the user during the submission process.

== Relationship between Sample, Fraction and MSRun

The relationship between the Sample and MSRun will be given by two other Properties:

* Label: It represents the information of a Multiplex or label-free experiment. It can be used to spot the type of experiment. For each Label two major characteristics are mandatory:

** Label Identifier: An identifier of the type of labeling in the Experiment Design Table.
** Label Name: A CvTerm with the type of Labeling applied: *["PRIDE", "PRIDE:0000312", "Label free", ]*
* Fraction Id

** Fraction Identifier: A unique fraction identifier to classify the sample.

*Additional*:

Some search engines as MaxQuant use the Fraction Group to perform better statistical analysis:

* Fraction Group
** Fraction Group Identifier: The fraction group identifier is used to classify multiple fractions.

*Examples Label Free Experiment*:

